gymnasium/wrappers/clip_action.py

```diff
@@ -25,6 +26,10 @@ def __init__(self, env: gym.Env):
         Args:
             env: The environment to apply the wrapper
         """
+        deprecation(
```

Comment:
Sorry about this, I don't think we want deprecation warnings yet. The way that Im thinking about it, is v0.27 includes the new wrappers, v0.28 includes the new vector env and vector wrappers then v0.29 will include the deprecation warnings. See https://github.com/Farama-Foundation/Gymnasium/pull/159 for more details on how we could do it

Reply:
No problem :wink: , removed in https://github.com/Farama-Foundation/Gymnasium/pull/150/commits/c869086369e7b95b068585f3ef9bec903d16b433